packages/state/recoil/selectors/misc.ts

```diff
@@ -0,0 +1,23 @@
+import { selectorFamily } from 'recoil'
+
+export const averageColorSelector = selectorFamily({
+  key: 'averageColor',
+  get: (url: string) => async () => {
+    if (!url || url.startsWith('http://localhost')) {
```

Comment:
What's going on here? Is this for when it's running on a development server? A comment would help here.